### ISSUE TITLE
Bundler.with_clean_env has been deprecated

### ISSUE THREAD

**Issue by tjwallace** [castwide/solargraph#252]
```
[DEPRECATED] `Bundler.with_clean_env` has been deprecated in favor of `Bundler.with_unbundled_env`. If you instead want the environment before bundler was originally loaded, use `Bundler.with_original_env` (called at /.../solargraph-0.38.0/lib/solargraph/documentor.rb:56)
```

**Comment by castwide** [comment]
Is this a warning or a fatal error? What versions of Bundler and Ruby are you using?

**Comment by tjwallace** [comment]
@castwide sorry, I should have included that info. It is just a warning.

* `Bundler version 2.1.0`
* `ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-darwin18]`

**Comment by castwide** [comment]
Thanks. This should be a minor issue. There's only one place where `Bundler.with_clean_env` gets called, and `with_unbound_env` should be a clean replacement when using Bundler 2.1+. I'll put this change in the pipeline for the next release.

**Comment by castwide** [comment]
Released in v0.38.2.

**Comment by sakshiseth** [comment]
This could be resolved if you check the version of the ruby in the GEMFILE is exactly the same as that of the installed version of ruby. If it mismatches then reinstall ruby with the version mentioned in the GEMFILE. This will eliminate the warning.
